api/src/db/migrations/versions/2024_03_12_add_indexes_for_search.py

```diff
@@ -0,0 +1,123 @@
+"""Add indexes for search
+
+Revision ID: f8364285630d
+Revises: 578c80acb29c
+Create Date: 2024-03-12 13:22:57.718265
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "f8364285630d"
+down_revision = "578c80acb29c"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_index(
+        op.f("current_opportunity_summary_opportunity_id_idx"),
+        "current_opportunity_summary",
+        ["opportunity_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("current_opportunity_summary_opportunity_status_id_idx"),
+        "current_opportunity_summary",
+        ["opportunity_status_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("current_opportunity_summary_opportunity_summary_id_idx"),
+        "current_opportunity_summary",
+        ["opportunity_summary_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("link_opportunity_summary_applicant_type_applicant_type_id_idx"),
+        "link_opportunity_summary_applicant_type",
+        ["applicant_type_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("link_opportunity_summary_applicant_type_opportunity_summary_id_idx"),
+        "link_opportunity_summary_applicant_type",
+        ["opportunity_summary_id"],
+        unique=False,
+    )
+    op.create_index(
+        op.f("link_opportunity_summary_funding_category_funding_category_id_idx"),
+        "link_opportunity_summary_funding_category",
+        ["funding_category_id"],
+        unique=False,
+    )
```

Comment:
Would any of the new indexes cause performance issues on write? Or I guess the opportunities are relatively stable and don't need bulk updates too often

Reply:
I wouldn't expect to see any issues, and adjusting the indexes is pretty whatever. From past experience, testing against a local DB versus one in AWS is a bit different in terms of performance. Can evaluate more once we have the data in the dev DB.

Testing locally with ~10k records, adding/removing the indexes is basically instant. Prod data will be about 80k, so let's just say about 10x. I don't think the scale of our DB is anywhere near "big data" to need to super optimize this, but if we do we can easily add/remove indexes uneventfully.